build/devbase/godeps.sh

```diff
@@ -39,8 +39,6 @@ github.com/cockroachdb/c-protobuf
 github.com/cockroachdb/c-rocksdb
 github.com/cockroachdb/c-snappy
 github.com/jteeuwen/go-bindata/go-bindata
-github.com/coreos/etcd/Godeps/_workspace/src/github.com/gogo/protobuf/proto
```

Comment:
These should stay. They're not actually important but in general this list should be identical to the output of `make listdeps`. We should also add golang.org/x/net/context to this list.